deploy/helm/prometheus-overrides.yaml

```diff
@@ -83,32 +83,32 @@ prometheus:
     - port: metrics
     namespaceSelector:
       matchNames:
-      - sumologic
+      - $(NAMESPACE)
     selector:
       matchLabels:
-        app: collection-sumologic-fluentd-logs
+        sumologic/app: fluentd-logs
   - name: collection-sumologic-fluentd-metrics
```

Comment:
Shoudl we leverage these on the ServiceMonitor name and labels (not the selector labels but the labels for the ServiceMonitor as well?

Reply:
What is the purpose of those labels? Do we use them anywhere in the sumologic?

Reply:
It’s more consistency. Should we be consistent about the label usage across all things we install.

Reply:
That makes sense, I read additionalLabels as "labels which will be added as dimension to the metric"